content/docs/minion-v0.4/environment.md

```diff
@@ -4,17 +4,24 @@ date = "2017-04-14T13:00:00+00:00"
 title = "Prepare the environment"
 parent = "/docs/minion-v0.4/"
 prev = "/docs/minion-v0.4/softwaremethodology/"
-next = "/docs/minion-v0.4/lowriscsetup/"
+next = "/docs/minion-v0.4/installtools/"
 showdisqus = true
 
 +++
 
-We still recommend you work with a 64-bit Ubuntu (14.04 LTS).
+Lowrisc tools and source code are robust between versions of Linux, but it has been
+found that
+Vivado (the Xilinx FPGA implementation suite) [is particular about the O/S](https://www.xilinx.com/support/answers/54242.html)
+
+All builds were completed with Vivado 2015.4. If a newer version is attempted there could be script incompatibilities to fix and/or incompatible upgrades to internal Xilinx IP. We have no evidence that newer versions do not work, however IP changes to support newer chip families can cause obscure error messages which are offputting for the first-time user.
+
+For this version the recommended O/S 64-bit Ubuntu (14.04 LTS). However, this is not stable on newer Intel chip sets, where 16.04 LTS should be used instead. If an unstable choice is made, the most likely result is LD_LIBRARY_PATH conflicts and/or synthesis crashes. Since Vivado implements a belt and braces approach, setting LD_LIBRARY_PATH and using a wrapper script, it should be safe to unset LD_LIBRARY_PATH in your shell. This will prevent conflicts with the obsolete libstc++ build-in to Vivado.
+
```

Comment:
I do not fully understand what do you mean "belt and braces approach", "using a wrapper script".